code/dictation.py

```diff
@@ -12,6 +12,21 @@
     desc="Look at surrounding text to improve auto-capitalization/spacing in dictation mode. By default, this works by selecting that text & copying it to the clipboard, so it may be slow or fail in some applications.",
 )
 
+mod.list("prose_modifiers", desc="Modifiers that can be used within prose")
+mod.list("prose_snippets", desc="Snippets that can be used within prose")
+ctx = Context()
+ctx.lists["user.prose_modifiers"] = ["cap", "no caps", "no space"]
+ctx.lists["user.prose_snippets"] = {
+    "spacebar": " ",
```

Comment:
Could use existing vocab features except I think they are an awkward fit for whitespace, and there is code here that interacts with these.

Reply:
Suppose you freshly enter dictation mode and you need to start with a space. This lets you add that. Conceptually, it is the opposite of "no space".